test/README.md

```diff
@@ -0,0 +1,205 @@
+# Official Images Test Suite
+
+## Running Tests
+
+```console
+$ ./run.sh
+
+usage: run.sh [-t test ...] [imageTag ...]
+   ie: run.sh debian:wheezy
+       run.sh -t utc python:3-onbuild
+       run.sh -t utc python:3-onbuild -t py-onbuild
+
+This script processes the specified Docker images to test their running
+environments.
+```
+
+Run all the tests that are applicable to the `debian:wheezy` image:
+
+```console
+$ ./run.sh debian:wheezy
+testing debian:wheezy
+	'utc' [1/2]...passed
+	'cve-2014--shellshock' [2/2]...passed
+```
+
+Try to run just the `python-hy` test against the `debian:wheezy` image: (which doesn't contain Python)
+
+```console
+$ ./run.sh -t python-hy debian:wheezy
+testing debian:wheezy
+```
+
+Run the `utc` test against both the `python:3` and `python:2` images:
+
+```console
+$ ./run.sh -t utc python:3 python:2
+testing python:3
+	'utc' [1/1]...passed
+testing python:2
+	'utc' [1/1]...passed
+```
+
+Run the `utc` and `python-hy` tests against `python:3`, `python:2`, and `debian:wheezy`:
+
+```console
+$ ./run.sh -t utc -t python-hy python:3 python:2 debian:wheezy
+testing python:3
+	'utc' [1/2]...passed
+	'python-hy' [2/2]...passed
+testing python:2
+	'utc' [1/2]...passed
+	'python-hy' [2/2]...passed
+testing debian:wheezy
+	'utc' [1/1]...passed
+```
+
+Run the `python-hy` test against `python:3` and `pypy:3`:
+
+```console
+$ ./run.sh -t python-hy python:3 pypy:3
+testing python:3
+	'python-hy' [1/1]...passed
+testing pypy:3
+	'python-hy' [1/1]...passed
+```
+
+## Writing Tests
+
+Each test lives in a separate directory inside `tests/`, and `config.sh` determines which images each test applies to.
+
+### `tests/<test name>/` *(anatomy of a test)*
+
+#### `tests/<test name>/run.sh`
```

Comment:
Yeah, markdown styling is annoying.  I'll just add a block of text in
between describing the directory in general.